_candidates_en/ujihisa_1.html

```diff
@@ -0,0 +1,22 @@
+---
+name: ujihisa
+title: 11 ways of legacy modernization with Scala
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: ujm
+github: ujihisa
+icon: https://avatars0.githubusercontent.com/u/11504?v=3&s=460
+organization: Hootsuite Media inc
+---
+
+<p>
+Hootsuite is a web service that was created in 2008, and since 2012 we have been using Scala for some parts of the system. In this session, I will share some of the lessons learned using one of the internal services we developed as an example.
```

Comment:
原文 : Hootsuite社は2008年に生まれたPHPのウェブサービスで、2012年から一部にScalaを使用しはじめました。2015年現在新機能の開発は原則Scalaで行うようにしています。
訳案 : Hootsuite is a web service with PHP that was created in 2008, and since 2012 we have been using Scala for some parts of the system. And currently, we basically use Scala for all our new features.

- PHPのという部分が抜けていた(がなくてもいいかも)
- 2015年現在の部分を加筆。今は `全部` Scalaという点は強調したいのではと思いましたので。